resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -42,6 +42,8 @@ RUN set -ex && \
     rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/*
 
 COPY jars /opt/spark/jars
+# Copy RELEASE file if exists
+COPY RELEAS[E] /opt/spark/RELEASE
```

Comment:
Why `[E]`?

Reply:
I used a trick glob pattern here in Dockerfile. Since `RELEASE` file doesn't exist in GitHub, `RELEAS[E]` matches `RELEASE` or `RELEAS` and this statement ignores when there is no such file.

Reply:
Thank you!